admin_guide/high_availability.adoc

```diff
@@ -14,34 +14,36 @@ toc::[]
-impose operational requirements to handle failure cases when one or more router
-instances are "unreachable".
-
-For some IP-based traffic services, virtual IP addresses (VIPs) should always be
-serviced for as long as a single instance is available. This simplifies the
-operational overhead and handles failure cases gracefully.
-
-IMPORTANT: Even though
-a service is highly available, performance can still be affected.
-
-Use cases for high-availability include:
-
-* I want my cluster to be assigned a resource set and I want the cluster to automatically manage those resources.
-* I want my cluster to be assigned a set of VIPs that the cluster manages and migrates (with zero or minimal downtime) on failure conditions, and I should not be required to perform any manual interactions to update the upstream "discovery" sources (e.g., DNS). The cluster should service all the assigned VIPs when at least a single node is available, despite the current available resources not being sufficient to reach the desired state.
+IP Failover uses link:http://www.keepalived.org/[*Keepalived*] to host a set of externally accessible
+Virtual IP (VIP) addresses on a set of hosts.  Each VIP is serviced by, at most, one of the hosts at
+any point in time.  *Keepalived* uses the VRRP protocol to determine which host in the set of hosts
+will service each VIP address.  If a host goes down or the service that *Keepalived* is watching goes down the
+VIP is serviced from another host in the set. As long as a single host is available the VIP is serviced.
+
+{product-title} supports an ipfailover Deployment Config (DC) created by the `oadm ipfailover` command. The ipfailover DC
+specifies the set of VIP addresses and the set of nodes on which to service them. The cluster can have
+multiple ipfailover DCs, each servicing its own set of VIP addresses. Each pod in the ipfailover DC runs
+*Keepalived* and each pod runs on a different node.
+
+When using VIPs to access a service, such as a router, the service should be running on all of the
+nodes that are running the ipfailover pods. This permits all ipfailover nodes to possibly service the VIPs and
+all service nodes to possibly be serviced. A miss match will result in either some ipfailover nodes never servicing
+the VIPs or some services to never receive traffic.  Using the same selector and replication count for both
+ipfailover and the service will eliminate the missmatch.
+
+The nodes in the high availibility set can come and go. The service on the node can start and stop.
+When a node or sevice becomes unavailable *keepalived* will select another node to service the VIP. The
+VIP will be serviced as long as at least one node with a running service is available.  As part of normal
+operation the VIP may be assigned to any of the nodes at any point in time.
+
+IMPORTANT: Even though a service VIP is highly available, performance can still be affected.
```

Comment:
affected by what?  We should say by load or something.  And that HA does not imply load balancing.